test/e2e/ginkgo/ginkgo_suite_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"flag"
+	"testing"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	ocmWorkClient "open-cluster-management.io/api/client/work/clientset/versioned"
+
+	"k8s.io/client-go/kubernetes"
+
+	ksClient "github.com/kubestellar/kubestellar/pkg/generated/clientset/versioned"
+	"github.com/kubestellar/kubestellar/test/util"
+)
+
+func TestGinkgo(t *testing.T) {
+	gomega.SetDefaultEventuallyTimeout(time.Minute)
+	gomega.RegisterFailHandler(ginkgo.Fail)
+	ginkgo.RunSpecs(t, "Kubestellar testing")
+}
+
+var (
+	ctx          context.Context
+	coreCluster  *kubernetes.Clientset
+	wds          *kubernetes.Clientset
+	ksWds        *ksClient.Clientset
+	wec1         *kubernetes.Clientset
+	wec2         *kubernetes.Clientset
+	ocmWorkImbs  *ocmWorkClient.Clientset
+	releasedFlag string
+)
+
+func init() {
+	flag.StringVar(&releasedFlag, "released", "", "released controls whether we use a release image")
+}
```

Comment:
How does a user control the debug log verbosity level (the thing that is normally set with `-v=N`)?

Reply:
I haven't found a way to pass a log verbosity level with ginkgo.  There is the '-v' flag for additional output, the '-vv' for maximum output, and the '-trace' to have a trace at each failure.  But I haven't found a logging level.  Perhaps we need to set that up as an external flag similar to -released?

Reply:
To be sure, this is not a blocker issue. The bash-based tests have fixed verbosity. As long as the verbosity is reasonably high --- high enough to debug problems --- that is good enough.